src/gc/gc.cpp

```diff
@@ -2083,9 +2083,9 @@ uint8_t* gc_heap::pad_for_alignment_large (uint8_t* newAlloc, int requiredAlignm
 #define END_SPACE_AFTER_GC (LARGE_OBJECT_SIZE + MAX_STRUCTALIGN)
 
 #ifdef BACKGROUND_GC
-#define SEGMENT_INITIAL_COMMIT (2*OS_PAGE_SIZE)
+#define SEGMENT_INITIAL_COMMIT (2*GCToOSInterface::GetPageSize())
```

Comment:
@sdmaclea ok, since it is 2:1 for the macro, let's use it then. While I prefer clarity to brevity, @Maoni0 is the one who maintains this code and so her preference is more important here.